src/lib/hover_thrust_estimator/ZeroOrderHoverThrustEkf.cpp

```diff
@@ -0,0 +1,94 @@
+#include "ZeroOrderHoverThrustEkf.hpp"
+#include <mathlib/mathlib.h>
+
+void ZeroOrderHoverThrustEkf::predict(const float dt)
+{
+	// State is constant
+	// Predict state covariance only
+	_P += _Q * dt;
```

Comment:
@bresch Don't you need to square it before adding?

Reply:
Yes, I also forgot to square R

Reply:
I'll square it in `setProcessNoise` and `setMeasurementNoise`

Reply:
@bresch Cool

Reply:
@bresch Hang on, I think you have to square the dt as well.

Reply:
I think you don't need to square dt.